### PR TITLE
No jQuery

### PR DESCRIPTION
**Pull request by dckesler**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dist": "webpack --optimize-minimize",
     "compile": "babel src --out-dir lib",
-    "build": "npm run dist && npm run compile"
+    "build": "npm run dist && npm run compile",
+		"prepublish": "npm run compile"
```
Comment (internettrans) [inline package.json]: not a huge deal, but npm is militant about no tabs in package.jsons. I usually cave to its demands

**Comment by internettrans** [comment]
So you removed the div that wraps the span? Why was that necessary for eradicating jquery?

**Comment by dckesler** [comment]
No it's still there just not in the same spot

**Comment by internettrans** [comment]
🚶

**Pull request merged**